releasenotes/notes/fix-tracer-agent-forwards-remote-config-errors-1e939be195e6db67.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    APM: Fixed trace-agent not forwarding errors from remote configuration and reporting them all as 500s
```

Optional non-blocking
```suggestion
    APM: Trace-agent forwards errors from remote configuration and reports them all as 404 instead of 500s.
```